### PR TITLE
Deprecate HDD0/disc, make RPCS3/games movable

### PR DESCRIPTION
**Pull request by elad335**
image
image

* Do not create /dev_hdd0/disc anymore, mounting /dev_bdvd inside /dev_hdd0 is wrong and creates a file reflection situation.
315mXpN

### CODE REVIEW

File: rpcs3/rpcs3qt/gui_settings.h [hunk 1]
```diff
@@ -190,6 +190,7 @@ namespace gui
 	const gui_save fs_dev_flash2_list   = gui_save(fs, "dev_flash2_list",   QStringList());
 	const gui_save fs_dev_flash3_list   = gui_save(fs, "dev_flash3_list",   QStringList());
 	const gui_save fs_dev_bdvd_list     = gui_save(fs, "dev_bdvd_list",     QStringList());
+	const gui_save fs_games_list        = gui_save(fs, "games_list",     QStringList());
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/gui_settings.h:193]: this alignment triggers my OCD xD

Comment (elad335) [reply rpcs3/rpcs3qt/gui_settings.h]: Fixed.

**Comment by Megamouse** [comment]
I guess it looks fine, although I don't understand what we need the disc dir still for

**Comment by elad335** [comment]
It's for good organization and auto-detection by RPCS3.

File: rpcs3/Emu/System.cpp [hunk 2]
```diff
@@ -408,7 +408,18 @@ void Emulator::Init(bool add_only)
...
-		make_path_verbose(dev_hdd0 + "disc/");
+		const std::string games_common_dir = g_cfg_vfs.get(g_cfg_vfs.games_dir, emu_dir);
+
+		if (make_path_verbose(games_common_dir))
+		{
+			fs::write_file(games_common_dir + "/Disc Games Can Be Put Here For Automatic Detection.txt", fs::create + fs::excl + fs::write, ""s);
+
+			if (std::string rpcs3_shortcuts = games_common_dir + "/shortcuts"; make_path_verbose(rpcs3_shortcuts))
+			{
+				fs::write_file(rpcs3_shortcuts + "/Copyable Shortcuts For Installed Games Would Be Added Here.txt", fs::create + fs::excl + fs::write, ""s);
```
Comment (Megamouse) [inline rpcs3/Emu/System.cpp:419]: as far as I can tell you aren't using the vfs in the shortcut creation, so this is a kinda weird file to put there

Comment (elad335) [reply rpcs3/Emu/System.cpp:420]: Fixed.

**Comment by Federock10** [comment]
Does anyone know if there's a Mac version that works for 10.0?

**Comment by AniLeo** [comment]
Locking due to spam

**Pull request merged**